builder/bundles.go

```diff
@@ -877,7 +878,40 @@ func createVersionsFile(baseDir string, packagerCmd []string) error {
 	return w.Flush()
 }
 
-func fixOSRelease(filename, version string) error {
+func fixOSRelease(b *Builder, filename, version string) error {
+
+	//Replace the default os-release file if customized os-release file found
+	path := b.Config.Mixer.OSReleasePath
+	if path != "" {
+
+		// Validate file path
+		_, err := os.Stat(path)
+		if err != nil {
+			return err
+		}
```

Comment:
Using Stat here can cause TOCTOU problems. There is no need to stat the file if you are going to open it afterwards since Open will error out jsut like stat if the file doesnt exist. Open by itself can perform both jobs here.

Reply:
No issue anymore since doesn't use copy operation.